modules/test/playwright/tests/dynamic-data-mapping-form-web/main/formSubmission.spec.ts

```diff
@@ -113,7 +115,7 @@ test.describe('Manage forms through submission page', () => {
 
 		await virtualInstanceFormBuilderPage.entriesTab.click();
 
-		await expect(virtualInstancePage.getByText('5379475')).toBeVisible();
+		await expect(virtualInstancePage.getByText('10353442')).toBeVisible();
```

Comment:
Change to
```
await expect(virtualInstancePage.getByText('Federative Republic of Brazil')).toBeVisible();
```

Reply:
Oh, of course, you don't have to use the official name property! If you prefer, you can import that lar, reconfigure the data provider, and use another (non-dynamic) property!